swagger/v1/swagger.yaml

```diff
@@ -103,14 +103,18 @@ paths:
           description: success
           content:
             application/json:
-              example:
-                smoke_test_one_result: true
+              examples:
+                example_0:
+                  value:
```

Comment:
Why has the non-live version updated too? 🤔

Reply:
I checked out main and ran `rake rswag:specs:swaggerize` and it changed it too. looks like it has not been run since 2021 and updates to rswag gems in 2022 may have changed its generation of the yaml. - just from looking at commit dates.

UI looks the same but it allows for multiple examples - the name of the example is `example_0` now. 

![Screenshot 2023-02-03 at 14 14 49](https://user-images.githubusercontent.com/7016425/216625515-80a78481-5c0a-4ad3-b4b6-65c3221db73e.png)